src/source/src/table_v2.rs

```diff
@@ -114,15 +114,15 @@ pub struct TableV2BatchReader;
 #[async_trait]
 impl BatchSourceReader for TableV2BatchReader {
     async fn open(&mut self) -> Result<()> {
-        unimplemented!()
+        Err(ErrorCode::NotImplemented("unimplemented BatchSourceReader".to_string(), None.into(), ).into())
```

Comment:
This struct will be removed soon, we can keep the `unimplemented!` or `panic!` here :)

Reply:
Done